### ISSUE TITLE
NullPointerException after fixing #144

### ISSUE THREAD

**Issue by mtheall** [eProsima/Fast-DDS-Gen#153]
This used to work before:
```cpp
const uint32 MAX_ITEMS = 6;

const uint32 TEST_VALUE = MAX_ITEMS;

struct test_t
{
	sequence<octet, MAX_ITEMS> test;
};
```
but now throws the following error:
```
error: Error evaluating array dimension: TEST_VALUE=MAX_ITEMS;MAX_ITEMS=6;(MAX_ITEMS) | 0
Exception in thread "main" java.lang.NullPointerException
	at com.eprosima.fastdds.fastddsgen.execute(fastddsgen.java:407)
	at com.eprosima.fastdds.fastddsgen.main(fastddsgen.java:1481)
```

**Comment by jsantiago-eProsima** [comment]
Hi @mtheall,

Thanks for the report. We have drafted https://github.com/eProsima/IDL-Parser/pull/65 with a tentative fix for this issue. Will let you know as soon as it's merged.

**Comment by JesusPoderoso** [comment]
Hi @mtheall 

The issue has been addressed and merged ( #155 ).
